server/test/routes/likes.test.js

```diff
@@ -53,30 +53,15 @@ describe('Testing likes routes /api/auth/:slug/like', () => {
   });
 
 
-  describe('if the user has not yet been verified tries to like an article', () => {
-    it('should return a 403(Unauthorized)', (done) => {
-      request(server)
-        .get(likeRoute)
-        .set('Authorization', unverifiedToken)
-        .end((err, res) => {
-          expect(res)
-            .property('status').to.equal(403);
-          expect(res.body)
-            .property('errors')
-            .property('message')
-            .to.equal('Your account has not been verified');
-          done();
-        });
-    });
-  });
+  
   describe('when like informations are retrieved when no one has liked the article', () => {
     it('should return a status code of 204 ', (done) => {
       request(server)
         .get(likeRoute)
         .set('Authorization', token)
         .end((err, res) => {
           expect(res)
-            .property('status').to.equal(204);
+            .property('status').to.equal(200);
```

Comment:
the it block says `should return a status code of 204` but the assertion is for 200, why ?

Reply:
I will edit the it block statement to should be 200 not 204 following our convention of returning 200 status for get routes even for empty results